pyface/data_view/abstract_value_type.py

```diff
@@ -19,11 +19,20 @@
 and how to actually display it.
 """
 
+from enum import IntEnum
+
 from traits.api import ABCHasStrictTraits, Event, observe
 
 from .data_view_errors import DataViewSetError
 
 
+class CheckState(IntEnum):
+    "Possible checkbox states"
+    # XXX in the future this may need a "partial" state
```

Comment:
I'm not sure what a partial state can mean for a checkbox.

Reply:
When you have a tree view a parent may show a 3rd "partially selected" state when some of the children are checked and some are not.  Visually it usually looks something like this:
![image](https://user-images.githubusercontent.com/600761/94020299-adc72a80-fdaa-11ea-8930-7a9386614879.png)

This is why I am using an enum here rather than simple bool.

Reply:
Can we put this enhancement into an issue and xref it in the code?